src/XmlDeserializationVisitor.php

```diff
@@ -79,6 +79,8 @@ public function prepare($data)
 
         $previousEntityLoaderState = null;
         if (\LIBXML_VERSION < 20900) {
+            // @todo resolve the deprecation
```

Comment:
It is already resolved by the `if` above.

Reply:
phpcs is complaining about the deprecated `libxml_disable_entity_loader` call. Should I just leave the `phpcs:ignore` there and remove the @todo line?

Reply:
It _is_ actually deprecated, but it is necessary for this library to run on distributions using older versions of libxml2. I think the only viable replacement here is to throw an exception, and I think this is best to propose in a dedicated PR.

Reply:
Done! I've left only the phpcs ignore line :)

Reply:
Thanks!